### PR TITLE
Add individual percentage per adult

### PR DESCRIPTION
**Pull request by adamgrad**

### CODE REVIEW

**Comment by adamgrad** [comment]
Diff for `content/reference/endpoints/fees.md` looks a bit scary but the only thing added there is:
```
maximum_possible_rate  | [Decimal](/reference/enums#formats) | Read       | Fee's maximum possible rate, calculated value won't exceed amount present on this attribute.
```
The rest is just alignment of first column to make it as wide as "individual_percentage_per_adult" to make table more readable.

File: content/reference/endpoints/fees.md [hunk 1]
```diff
@@ -11,41 +11,43 @@
...
-updated_at        | [Time](/reference/enums#formats) | Read         | Fee's update time.
+exchange_rate          | [Decimal](/reference/enums#formats) | Read       | The rate used to convert from the `original_currency` to `currency`.
+name                   | [Object](/reference/enums#formats)   | Read/Write | ** Required** Fee's name, list of locales described in [enums section](/reference/enums#locales).
+kind                   | String  | Read/Write | Fee's kind, list of fee's kinds described in [enums section](/reference/enums#fee-kinds)
+original_currency      | String  | Read       | Fee's original currency, list of currencies codes described in [enums section](/reference/enums#currencies)
+rate                   | [Decimal](/reference/enums#formats)  | Read/Write |  ** Required** Fee's rate.
+rate_kind              | String  | Read/Write |  ** Required** Fee's rate kind, list of fee's rate kinds described in [enums section](/reference/enums#fee-rate-kinds).
+maximum_possible_rate  | [Decimal](/reference/enums#formats) | Read/Write | Fee's maximum possible rate, calculated value won't exceed amount present on this attribute.
```
Comment (Azdaroth) [inline content/reference/endpoints/fees.md]: this is used only for inidivual percentage per adult, no? good to mention that

Comment (adamgrad) [reply content/reference/endpoints/fees.md]: not really. It's used for all "percentage" attributes so I'll add info about it

**Comment by adamgrad** [comment]
@Azdaroth feedback applied.

**Pull request merged**